cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -33,7 +34,11 @@ import (
 	kubearchivev1alpha1 "github.com/kubearchive/kubearchive/cmd/operator/api/v1alpha1"
 )
 
-const SinkFilterConfigMapName = "sink-filters"
+const (
+	SinkFilterConfigMapName   = "sink-filters"
+	ApiServerSourceLabelName  = "kubearchive.org/enabled"
+	ApiServerSourceLabelValue = "true"
+)
 
 var k9eNs = os.Getenv("KUBEARCHIVE_NAMESPACE")
```

Comment:
Are we officially using `k9e` as an abbreviation of KubeArchive in the code?

Reply:
> Are we officially using k9e as an abbreviation of KubeArchive in the code?

I don't think there's anything official, but I think it makes this code clearer.  It was done as part of a change that was merged over a month ago, and is documented at the top of the file.

>  I think "ka" makes much more sense.

You say "toe-may-toe" and I  say "toe-ma-toe".